src/q4/opti-INMA1702/notes/cm/cm1-problemes.tex

```diff
@@ -156,55 +156,59 @@ \subsection{Problème de flots (ou de transport)}
+	Cela implique qu'on n'a plus le même problême que dans le cas continu.
 
 \subsection{Problème d'affectation}
+\label{sec:affectation}
 
-\begin{center}
-\begin{tikzpicture}
-	\node [rectangle] (pers) at (-3,0) {personnes ($i$)};
-	\node [rectangle] (taches) at (0,2) {tâches ($j$)};
-
-	\draw[step=0.5cm,color=gray] (-1,-1) grid (1,1);
-		\node at (-0.75,+0.75) {1};
-		\node at (-0.25,+0.75) {0};
-		\node at (+0.25,+0.75) {0};
-		\node at (+0.75,+0.75) {0};
-		\node at (-0.75,+0.25) {0};
-		\node at (-0.25,+0.25) {0};
-		\node at (+0.25,+0.25) {1};
-		\node at (+0.75,+0.25) {0};
-		\node at (-0.75,-0.25) {0};
-		\node at (-0.25,-0.25) {1};
-		\node at (+0.25,-0.25) {0};
-		\node at (+0.75,-0.25) {0};
-		\node at (-0.75,-0.75) {0};
-		\node at (-0.25,-0.75) {0};
-		\node at (+0.25,-0.75) {0};
-		\node at (+0.75,-0.75) {1};
-\end{tikzpicture}
-\end{center}
-
-Le problème d'affectation revient à dire
-qu'on a un certain nombre de personnes
-qui peuvent chacune faire une parmi plusieurs tâches.
-La façon la plus efficace de représenter ce problème est à l'aide d'une matrice
-(appelée matrice d'affectation) qui dit si oui ou non une personne $i$
-fait la tâche $j$.
-Chaque tâche à un certain coût $p_i$ pour la personne $i$.
-Si la personne $i$ fait la tâche $j$, $x_{ij}$ vaut 1,
-sinon il vaut 0.
-Sous forme mathématique, on a
-
-
-\begin{equation*}
-\begin{array}{c@{\quad} r c l@{\qquad} l}
-	\min\limits_{x} & \sum\limits_{{\displaystyle (i,j) \in \mathcal{E}}} p_i x_{ij} &&&\\
-	\textnormal{s.t.} & x_{ij} & \in & \{0, 1\} & \forall i,j.
-\end{array}
-\end{equation*}
-
-où $\mathcal{E}$ est l'ensemble des paires (personne, tâche) possibles.
-Le problème d'affectation est donc a priori un problème de flots discret.
+	\begin{center}
+	\begin{tikzpicture}
+		\node [rectangle] (pers) at (-2.3,0) {personnes ($i$)};
+		\node [rectangle] (taches) at (0,1.5) {tâches ($j$)};
+
+		\draw[step=0.5cm,color=gray] (-1,-1) grid (1,1);
+			\node at (-0.75,+0.75) {1};
+			\node at (-0.25,+0.75) {0};
+			\node at (+0.25,+0.75) {0};
+			\node at (+0.75,+0.75) {0};
+			\node at (-0.75,+0.25) {0};
+			\node at (-0.25,+0.25) {0};
+			\node at (+0.25,+0.25) {1};
+			\node at (+0.75,+0.25) {0};
+			\node at (-0.75,-0.25) {0};
+			\node at (-0.25,-0.25) {1};
+			\node at (+0.25,-0.25) {0};
+			\node at (+0.75,-0.25) {0};
+			\node at (-0.75,-0.75) {0};
+			\node at (-0.25,-0.75) {0};
+			\node at (+0.25,-0.75) {0};
+			\node at (+0.75,-0.75) {1};
+	\end{tikzpicture}
+	\end{center}
+
+	Le problème d'affectation revient à dire
+	qu'on a un certain nombre de personnes
+	qui peuvent chacune faire une parmi plusieurs tâches.
+	La façon la plus efficace de représenter ce problème
+	est à l'aide d'une matrice (appelée matrice d'affectation)
+	qui dit si oui ou non une personne $i$
+	fait la tâche $j\,$.
+	Chaque tâche à un certain coût $p_i$ pour la personne $i\,$.
+	Si la personne $i$ fait la tâche $j$, $x_{ij}$ vaut 1,
+	sinon il vaut 0.
+	Sous forme mathématique, on a
+
+
```

Comment:
2 line breaks?

Reply:
Here, it's no use since the next part is an equation, but usually 3 line breaks makes for 2 blank lines and thus a new paragraph.